### PR TITLE
Fix build codebase inconsistencies

### PR DESCRIPTION
**Pull request by edisonguo**
@bje- Today when I was trying to build a docker image for gsky using ./configure and make instead of the ansible scripts we internally used, I found there are a few places in the codebase that are not consistent with the Makefile script. This PR fixes them.

### CODE REVIEW

File: ows.go [hunk 1]
```diff
@@ -182,7 +183,7 @@ func serveWMS(ctx context.Context, params utils.WMSParams, conf *utils.Config, r
 
 		xRes := (params.BBox[2] - params.BBox[0]) / float64(*params.Width)
 		if conf.Layers[idx].ZoomLimit != 0.0 && xRes > conf.Layers[idx].ZoomLimit {
-			out, err := utils.GetEmptyTile("zoom.png", *params.Height, *params.Width)
+			out, err := utils.GetEmptyTile(utils.EtcDir+"/zoom.png", *params.Height, *params.Width)
```
Comment (bje-) [inline ows.go]: I would not put the `.png` files in `utils.EtcDir` (that's for configuration), but in `utils.DataDir`. The `share` directory is for architecture-independent data (and can be shared across architectures in a big, site-wide NFS mounted `/usr/local` file system, eg. `/usr/local/share/emacs`).

File: ows.go [hunk 2]
```diff
@@ -232,7 +233,7 @@ func serveWMS(ctx context.Context, params utils.WMSParams, conf *utils.Config, r
 			}
 
 			if norm[0].Width == 0 || norm[0].Height == 0 {
-				out, err := utils.GetEmptyTile("data_unavailable.png", *params.Height, *params.Width)
+				out, err := utils.GetEmptyTile(utils.EtcDir+"/data_unavailable.png", *params.Height, *params.Width)
```
Comment (bje-) [inline ows.go]: Likewise.

File: processor/tile_jpg_enc.go [hunk 3]
```diff
@@ -45,7 +46,7 @@ func (enc *JPGEncoder) Run() {
 	case 1:
 		buf := new(bytes.Buffer)
 		if nameSpaces[0] == "OutOfZoom" {
-			f, _ := os.Open("zoom.png")
+			f, _ := os.Open(utils.EtcDir + "/zoom.png")
```
Comment (bje-) [inline processor/tile_jpg_enc.go]: Likewise.

File: processor/tile_png_enc.go [hunk 4]
```diff
@@ -62,7 +63,7 @@ func (enc *PNGEncoder) Run() {
 	case 1:
 		buf := new(bytes.Buffer)
 		if nameSpaces[0] == "OutOfZoom" {
-			f, err := os.Open("zoom.png")
+			f, err := os.Open(utils.EtcDir + "zoom.png")
```
Comment (bje-) [inline processor/tile_png_enc.go]: Likewise.

File: processor/tile_png_enc.go [hunk 5]
```diff
@@ -134,7 +135,7 @@ func (enc *PNGEncoder) Run() {
 	default:
 		log.Printf("Cannot encode other than 1 or 3 namespaces into a PNG. Received %d namespaces: %v\n", len(nameSpaces), nameSpaces)
 		buf := new(bytes.Buffer)
-		f, err := os.Open("data_unavailable.png")
+		f, err := os.Open(utils.EtcDir + "/data_unavailable.png")
```
Comment (bje-) [inline processor/tile_png_enc.go]: Likewise.

**Comment by bje-** [comment]
@edisonguo This should probably be applied to the 1.0 branch, yes?

**Comment by edisonguo** [comment]
yes this should definitely be applied to v1.0.

**Comment by bje-** [comment]
OK, I will cherrypick the patch onto `v1.0` once I merge this PR.

**Pull request merged**